src/content.js

```diff
@@ -594,6 +602,11 @@ function init(options) {
 
 			if (pageDetect.isMilestone()) {
 				addMilestoneNavigation();
+				sortMileStonesByClosestDueDate();
+			}
+
+			if (pageDetect.isMilestoneList() || pageDetect.isLabelList() || pageDetect.isLabel()) {
+				sortMileStonesByClosestDueDate();
```

Comment:
Better to have one if statement with all of these `pageDetect`s than spreading `sortMileStonesByClosestDueDate()` in multiple different ones.